docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,149 @@
+4. You're well on your way to expense automation freedom!
+   
+## Default Categories based on specific MCC codes
+
+If you require more granular detail, the MCC Editor gives you even greater control over which MCC Codes are assigned to which Categories. The MCC Editor can be found just below the Default Categories table.
+
+## Implicit categorization
+Over time, Expensify will learn how you categorize certain merchants and then automatically apply that category to the same merchant in the future.
+
+- You can always change the category; we'll try to remember that correction for next time!
+- Any Expense Rules you have set up will always take precedence over implicit categories. 
+- Implicit categorization will **only** apply to expenses if you have **not** explicitly set a category already.  Changing the category on one expense does not change it for any other expense that has an explicit category already assigned.
+  
+This built-in feature will only use the categories from the currently active workspace on your account. You can change the active workspace by clicking your account icon in the app and selecting the correct workspace name before you SmartScan.
+  
+## Category violations
+Category violations can happen for the following reasons:
+
+- An employee categorized an expense with a category not included in the workspace's categories. This would throw a "category out of workspace" violation
+- If you change your categories importing from an accounting integration, this can cause an old category to still be in use on an open report which would throw a violation on submission. Simply reselect a proper category to clear violation.
+
+
+If Scheduled Submit is enabled on a workspace, expenses with category violations will not be auto-submitted unless the expense has a comment added.
+
+
+# FAQ
+## The correct category list isn't showing when one of my employees is categorizing their expenses. Why is this happening?
+Its possible the employee is defaulted to their personal workspace so the expenses are not pulling the correct categories to choose from. Check to be sure the report is listed under the correct workspace by looking under the Details section on top right of report. 
+## Will the account numbers from our accounting system (QuickBooks Online, Sage Intacct, etc.) show in the Category list when employees are choosing what chart of accounts category to code their expense to?
+The GL account numbers will be visible in the policy settings when connected to a Control-level policy for Policy Admins to see. We do not provide this information in an employee-facing capacity because most employees do not have access to that information within the accounting integration. 
```

Comment:
```suggestion
The GL account numbers will be visible in the workspace settings when connected to a Control-level workspace for workspace admins to see. We do not provide this information in an employee-facing capacity because most employees do not have access to that information within the accounting integration. 
```